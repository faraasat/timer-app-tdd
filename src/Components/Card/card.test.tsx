import { shallow } from "enzyme";
import CardComponent from "./card.component";

describe("Card Component Shallow", () => {
  let shallowRender = shallow(<CardComponent children={<div></div>} />);

  it("should match snapshot", () => {
    expect(shallowRender.debug()).toMatchSnapshot();
  });

  it("must have one div", () => {
    expect(shallowRender.find("div").length).toBe(2);
  });

  it("should have class timer-card-component", () => {
    expect(shallowRender.find(".timer-card-component").length).toEqual(1);
  });
});
