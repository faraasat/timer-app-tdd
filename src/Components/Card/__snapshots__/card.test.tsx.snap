// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component Shallow should match snapshot 1`] = `
"<div className=\\"timer-card-component\\">
  <div />
</div>"
`;
